<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 224, 224);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_singlemode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>480</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>54</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>지뢰찾기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_singlesetting">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>195</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string>[  게임 설정  ]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_singlestart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>135</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border : 0px;
</string>
    </property>
    <property name="text">
     <string>[  게임 시작  ]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
